packages/vue/src/components/organisms/SfHeader/SfHeader.js

```diff
@@ -60,5 +64,48 @@ export default {
         return ["", "account", "wishlist", "cart"].includes(value);
       }
     }
+  },
+  data() {
+    return {
+      isVisible: true,
+      isScrolling: false,
+      scrollTop: 0,
+      lastScrollTop: 0
+    };
+  },
+  beforeMount() {
+    window.addEventListener(
```

Comment:
If you register event please unregister it on `beforeDestory`
If `sticky=false` you shouldn't fire sticky functions :)